docs/source/installed_packages.rst

```diff
@@ -0,0 +1,44 @@
+Using Installed Packages
+========================
+
+
+Making PEP 561 compatible packages
+**********************************
+
+Packages that supply type information should put a ``py.typed`` in their package
+directory. For example, with a directory structure as follows:
```

Comment:
The pyi/py resolution is treated the same as a normal package on the `MYPYPATH`, so I would find that redundant. However, if you think it is relevant and valuable I will add it.

Reply:
> The pyi/py resolution is treated the same as a normal package on the MYPYPATH, so I would find that redundant. However, if you think it is relevant and valuable I will add it.

It would make it clearer to users that such packages can also supply .pyi files.